airflow/cli/commands/maintenance_command.py

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Maintenance sub-commands"""
+from airflow.utils.metastore_cleanup import run_cleanup
+
+
+def cleanup(args):
+    """Purges old records in metastore database"""
+    print(args.dry_run)
+    kwargs = dict(
+        dry_run=args.dry_run, clean_before_timestamp=args.clean_before_timestamp, verbose=args.verbose
+    )
+    if args.tables:
+        kwargs.update(
+            table_names=args.tables,
+        )
+    run_cleanup(**kwargs)
```

Comment:
Since you do `if table_name` in `run_cleanup`, this is equivalent to

```python
run_cleanup(
    dry_run=args.dry_run,
    clean_before_timestamp=args.clean_before_timestamp,
    verbose=args.verbose,
    table_names=args.tables,
)
```

With appropriate docstring in `run_cleanup` (to clarify that passing an empty list is the same as not passing the argument), this would be more readable and maintainable that the current implementation IMO.